oscarlab/graphene
# [Pal/Linux-SGX] Make the pal-sgx-sign code importable

## Affected components


## Description of the changes 

This moves the Python implementation of pal-sgx-sign into pal_sgx_sign.py
and replaces the cmdline tool with a tiny wrapper. This allows other
code to import it as a module and thereby re-use it's code.

Designing an API and then putting it into a proper Python package is left
for the future.

## How to test this PR? 

Signing manifests for SGX should work as before. Additionally you can now `import pal_sgx_sign` in Python assuming its dir is in `sys.path`.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/1296)
